### PR TITLE
feat: validate language code

### PR DESCRIPTION
**Pull request by will-larkin-nhs**
## Description
- new language codes table with a list of valid language codes
- validation for preferred language to check if the code is in the table

## Context
[DTOSS-3764](https://nhsd-jira.digital.nhs.uk/browse/DTOSS-3764)

## Type of changes

- [ ] Refactoring (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ 
- [x] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: application/CohortManager/src/Functions/ScreeningValidationService/LookupValidation/LookupValidation.cs [hunk 1]
```diff
@@ -55,6 +55,8 @@ public async Task<HttpResponseData> RunAsync([HttpTrigger(AuthorizationLevel.Ano
         {
             var existingParticipant = requestBody.ExistingParticipant;
             newParticipant = requestBody.NewParticipant;
+            System.Console.WriteLine("Language code: " + newParticipant.PreferredLanguage);
```
Comment (andrewmorris43) [inline application/CohortManager/src/Functions/ScreeningValidationService/LookupValidation/LookupValidation.cs:58]: Can these console logs be removed

File: tests/ScreeningValidationServiceTests/LookupValidation/LookupValidationTests.cs [hunk 2]
```diff
@@ -14,6 +14,7 @@ namespace NHS.CohortManager.Tests.ScreeningValidationServiceTests;
 using NHS.CohortManager.ScreeningValidationService;
 using RulesEngine.Models;
 using Data.Database;
+using Grpc.Core;
```
Comment (andrewmorris43) [inline tests/ScreeningValidationServiceTests/LookupValidation/LookupValidationTests.cs:17]: Not sure what this using statement is for?

Comment (will-larkin-nhs) [reply tests/ScreeningValidationServiceTests/LookupValidation/LookupValidationTests.cs]: removed

File: application/CohortManager/src/Functions/ScreeningValidationService/LookupValidation/LookupValidation.cs (hunk 1 above)
Comment (will-larkin-nhs) [reply application/CohortManager/src/Functions/ScreeningValidationService/LookupValidation/LookupValidation.cs]: removed

**Review by andrewmorris43 — APPROVED** [review]

**Review by Warren-Pitterson — APPROVED** [review]

**Pull request merged**